microsoft-365/security/defender-endpoint/linux-updates.md

```diff
@@ -68,7 +68,7 @@ sudo apt-get install --only-upgrade mdatp
 ```
 
 > [!IMPORTANT]
-> When integrating Microsoft Defender for Endpoint and Defender for Cloud, the mdatp agent will automatically receive updates by default.
+> When Defender for Cloud is provisioning the Microsoft Defender for Endpoint agent to Linux Servers, it will keep the client updated automatically.
```

Comment:
```suggestion
> When Defender for Cloud is provisioning the Microsoft Defender for Endpoint agent to Linux servers, it will keep the client updated automatically.
```